components/menu/OverflowMenu/style.module.scss

```diff
@@ -6,36 +6,47 @@
 }
 
 .overflow-menu__button {
-  @include circle(26px);
+  @include circle(36px);
   padding: $size-md;
   border: none;
   background-color: $color-gray;
   display: inline-block;
   cursor: pointer;
   position: relative;
+  box-shadow: inset 0 1px 0 0 rgba(0, 0, 0, 0.08);
 
   &:focus {
     outline: 0;
   }
 }
 
-.overflow-menu__svg {
+.overflow-menu__svg,
+.overflow-menu__svg--opened {
   @include completely-center();
-  width: 40%;
-  height: 60%;
+  width: 36.111111111%;
+  height: 44.444444444%;
```

Comment:
These look like magic numbers. What are they for?

Reply:
They control the ellipsis size. This allows them to scale relatively with the overall overflow button. I could hard code them to be 13px by 16px, but if the circle gets smaller or bigger, it'll lock the ellipsis size.